src/checkbox.test.miscellaneous.ts

```diff
@@ -12,12 +14,13 @@ test('registers itself', { tag: '@miscellaneous' }, async ({ mount, page }) => {
 
 test('can be disabled', { tag: '@miscellaneous' }, async ({ mount, page }) => {
   await mount(
-    () => html`<glide-core-checkbox label="Label" disabled></glide-core-checkbox>`,
+    () =>
+      html`<glide-core-checkbox label="Label" disabled></glide-core-checkbox>`,
   );
 
   const host = page.locator('glide-core-checkbox');
 
-  expect(host.waitFor).rejects.toThrow();
+  await expect(host.waitFor).rejects.toThrow();
```

Comment:
Need this (and a few other similar places) after updating `eslint.config.js` below